src/NServiceBus.Core/Licensing/LicenseManager.cs

```diff
@@ -24,6 +29,12 @@ internal void InitializeLicense(string licenseText)
 
             if (string.IsNullOrWhiteSpace(licenseText))
             {
+                // Display the trial start dialog.
```

Comment:
@SimonCropp - my bad. that comment needs to be deleted. At one point I had a dialog that would be displayed.